app/models/govbox/api_connection_with_obo_support.rb

```diff
@@ -27,6 +27,6 @@ def destroy_with_box?
 
   def validate_box(box)
     box.errors.add(:settings_obo, :not_allowed) if obo.present?
-    box.errors.add(:settings_obo, :invalid) if boxes.where.not(id: box.id).where("settings ->> 'obo' = ?", box.settings_obo).exists?
+    box.errors.add(:settings_obo, :invalid) if boxes.where.not(id: box.id).map { |b| b.settings_obo.to_s }.include?(box.settings_obo.to_s)
```

Comment:
Tu som skor za to, ze upravme tu logiku ze prazdny string nedovolime a bude tam vzdy nil lebo akonahle povolime prazdny string tak sa tato podmienka rozlezie aj vsade inde. Musime cistit data najskor ako sa len daju.